src/org/usfirst/frc/team4571/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package org.usfirst.frc.team4571.robot.subsystems;
+
+import java.util.logging.Logger;
+
+import org.usfirst.frc.team4571.robot.RobotConstants;
+import org.usfirst.frc.team4571.robot.subsystems.PID.DistanceOutput;
+import org.usfirst.frc.team4571.robot.subsystems.sensors.EncoderAverage;
+
+import com.ctre.CANTalon;
+
+import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class IntakeSubsystem extends Subsystem {
+	private CANTalon rightMotor;
+	private boolean isRollerIn;
+	public static final DoubleSolenoid ROLLER_SOLENOID = new DoubleSolenoid(RobotConstants.ROLLER_FOWARD_SOLENOID_CHANNEL, RobotConstants.ROLLER_REVERSE_SOLENOID_CHANNEL);
+
+	public IntakeSubsystem(){
+		this.rightMotor = new CANTalon(RobotConstants.RIGHT_MOTOR_CHANNEL);
+
+	}
+	// Put methods for controlling this subsystem
+	// here. Call these from Commands.
+
+	public void initDefaultCommand() {
+		// Set the default command for a subsystem here.
+		//setDefaultCommand(new MySpecialCommand());
+	}
+
+	public void initialize() {
+		out(0);
+	}
+	public void out(double speed){
+		ROLLER_SOLENOID.set(DoubleSolenoid.Value.kForward);
+		this.rightMotor.set(speed);
```

Comment:
The "rightMotor" value is set through your "forward" method, so it shouldn't be set here (you would want to call the "forward" method separately if you do). This would also mean that "out" does not need a speed value.